core-services/ci-chat-bot/workflows-config.yaml

```diff
@@ -168,6 +168,14 @@ workflows:
     platform: aws
   cucushift-installer-rehearse-azure-ipi:
     platform: azure
+  cucushift-installer-rehearse-azure-ipi-ovn-winc:
+        tag: upi-installer
```

Comment:
Looks like this line is unexpected

Reply:
Apologies for that... Fixing and pushing